### PR TITLE
onefetch 2.19.0

### PR DESCRIPTION
**Pull request by o2sh**
Created by https://github.com/mislav/bump-homebrew-formula-action

### CODE REVIEW

**Comment by chenrui333** [comment]
`    Minitest::Assertion: No linkage with libgit2! Cargo is likely using a vendored version.`

**Comment by chenrui333** [comment]
- relates to https://github.com/o2sh/onefetch/pull/1163

File: Formula/o/onefetch.rb [hunk 1]
```diff
@@ -21,21 +18,18 @@ class Onefetch < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "2e8c88c3adf425e057cf7a0797d2c94d92a974f4bde6389848d9bf22b33615e3"
   end
 
-  # `cmake` is used to build `zstd` and `zlib`.
-  # TODO: See if it is possible to use Homebrew dependencies instead.
+  # `cmake` is used to build `zlib`.
+  # upstream issue, https://github.com/rust-lang/libz-sys/issues/147
```
Comment (chenrui333) [inline Formula/o/onefetch.rb:22]: - https://github.com/rust-lang/libz-sys/issues/147

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]
Should probably add a linkage check for `zstd` in a follow up PR.

**Comment by chenrui333** [comment]
> Should probably add a linkage check for `zstd` in a follow up PR.

it is weird that in the linkage check, there is no zstd refs :(

**Pull request merged**